hypervisor/arch/x86/guest/vcpuid.c

```diff
@@ -675,6 +678,7 @@ void guest_cpuid(struct acrn_vcpu *vcpu, uint32_t *eax, uint32_t *ebx, uint32_t
 			break;
 
 		case 0x0bU:
+		case 0x1fU:
```

Comment:
0x1f uses 0b leave?

Reply:
Now I use CPUID.0b to emulate CPUID.1f for: Our platform doesn't support Level type: "3: Module, 4: Tile, 5: Die"